resources/metadata/detect.yaml

```diff
@@ -110,6 +111,35 @@ spec:
         - PARAMETERS
         - STAGES
         - STEPS
+      - name: groups
+        description: Users groups to be assigned for the Project
+        aliases:
+          - name: detect/groups
+        type: string
```

Comment:
can only be one group assigned?
If not, we should make this multi-value -> `[]string`

If only one group can be assigned the parameter should rather be `group`.

Maybe some more explanation would be helpful in the description.